databroker/core.py

```diff
@@ -1443,6 +1460,50 @@ def parse_handler_registry(handler_registry):
     return result
 
 
+def load_transforms(transforms):
```

Comment:
Minor preference for `parse_transforms` for symmetry with `parse_handler_registry`. We may or may not be actually loading anything, depending on the user input.

Reply:
updated